Dosbodoke/festival-chooselife-app
# 8 update next intl version

# What is the purpose of this Pull Request?

Bump `next-intl` to V3 Release Candidate so it can be used whiting server components

# What was made to achieve it?

# How to test if it works?

- Select `portugese` as default language
- Go to the home page
- Create a new Highline
- Go to the Highline page
- go back to home page

All previous steps should be translated in Portuguese and the URL should not have a prefix `/pt`

Nou change to english, repeat the steps. The only difference is that the URL should have a `/en` prefix. E.g.: `/en/[highline_id]`

# This PR can affect other functionalities? Which?
